metaflow/plugins/pypi/parsers.py

```diff
@@ -28,7 +33,7 @@ def requirements_txt_parser(content: str):
 
     Raises
     ------
-    ValueError
+     ParserValueError
```

Comment:
any reason to have a custom error? ideally it's nice to just rely on a built-in error